.idea/kotlinc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I added `.idea/` to `.gitignore` under the idea that IDE settings should be one's own (plus, I think this file might be partially or fully auto-generated).  Can we remove this file entirely from the online repo?

Reply:
I'm not sure about this. Those folks, who are working in IDEA must receive this files to re-store environment